src/capi/azext_capi/_help.py

```diff
@@ -141,3 +141,11 @@
 type: command
 short-summary: Update a CAPI management cluster.
 """
+
+helps['capi install'] = """
+type: command
+short-summary: installs all needed tools
+parameters:
+  - name: --all_tools
```

Comment:
If we are specifying the options_list in `_params.py` for `capi install` as `--all -a` need to refer in same way here to reflect changes in CLI  
```suggestion
  - name: --all -a
```